### ISSUE TITLE
VPAT Output

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#483]
## Problem
Taking Equlaify data into a VPAT requires a lot of work. Since VPATs are a standard template for accessibility pros, Equalify data should be able to quickly be added to a VPAT.

## Solution
There are three possible ways to display Equalify data as a VPAT:

1. Build a mini-app that uses the Equalify API to parse data into a CSV VPAT format
2. Update the Equalify API to output data as a VPAT
3. Update the frontend, adding a "VPAT" report feature.

I think we can quickly test this work with a mini app, then move onto deeper integration with Equalify as we see how people use the mini app. The miniapp would just pre-fill a CSV with information that can be gleaned from automated tests.

## Process
I'm interested in building the mini app, with help from @hmig.

It would be really helpful if @hmig could provide some samples of ideal outputs that we tool the app to realize. I'm attaching a sample VPAT with a table that we could use. Curious if there are any other examples to refrence.
[Sample-VPAT.pdf](https://github.com/user-attachments/files/17562708/Sample-VPAT.pdf)

**Comment by bbertucc** [comment]
@hmig (or anyone else)- I have some time to do open source contribution on this on November 13, but feel free to jump on it early if you want. We could probably just use the [get/results](https://docs.equalify.app/#016c49c6-a2fe-43e0-98e0-0893119d73d6) endpoint to output json that the app then reformats into a CSV.

**Comment by bbertucc** [comment]
Since we're focusing on #500 i'm going to close this. But feel free to re-open!
